sdk/nodejs/storage/accessTier.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export module AccessTiers {
+    export const BlobStorage: AccessTier = "BlobStorage";
+    export const StorageV2:   AccessTier = "StorageV2";
+}
+
+export type AccessTier =
+    "BlobStorage" |
+    "StorageV2"   ;
```

Comment:
AFAIK `Access Tier` is something like `Hot` or `Cold`

Reply:
I took these directly from the Terraform provider docs...

Reply:
They got you tricked: 
```
Defines the access tier for `BlobStorage` and `StorageV2` accounts. 
Valid options are `Hot` and `Cool`, defaults to `Hot`.
```